client/my-sites/site-settings/style.scss

```diff
@@ -479,6 +479,11 @@ button.site-settings__general-settings-set-guessed-timezone.button.is-borderless
 		box-shadow: none;
 	}
 }
+
+.site-settings .site-settings__wordpress-version {
+	display: inline;
```

Comment:
This probably isn't needed for `strong` tag anymore?

Reply:
Yep, good call!